pom.xml

```diff
@@ -94,7 +94,7 @@
     <regexp-version>1.4</regexp-version>
     <rome-version>2.1.0</rome-version>
     <shiro-version>1.13.0</shiro-version>
-    <slf4j-version>2.0.9</slf4j-version>
+    <slf4j-version>2.0.11</slf4j-version>
```

Comment:
I have to double check the package version in the OSGi/Karaf part. It should be OK but I would like to double check (as I did some other updates).